CHANGELOG.md

```diff
@@ -12,12 +12,15 @@ This changelog is a work in progress and may contain notes for versions which ha
 
 - Implemented custom order filters, which allow users to filter out all but the orders they care about. When a custom order filter is specified, Mesh will only send and receive orders that pass the filter. ([#630](https://github.com/0xProject/0x-mesh/pull/630)).
 - Developers can now override the contract addresses for any testnet using the `CUSTOM_CONTRACT_ADDRESSES` env config ([#640](https://github.com/0xProject/0x-mesh/pull/640)).
+- Added support for passing in your own Web3 provider when using the `@0x/mesh-browser` package. ([#665](https://github.com/0xProject/0x-mesh/pull/665)).
+
+## v8.2.0
+
+### Features ✅
+
 - Added `getOrdersForPageAsync` method to `@0x/mesh-rpc-client` WS client interface so that clients can paginate through the retrieved orders themselves ([#642](https://github.com/0xProject/0x-mesh/pull/642)).
```

Comment:
@fabioberger AFAICT this wasn't actually included in https://github.com/0xProject/0x-mesh/pull/676. Did you mean to include it?

Reply:
Good catch. Moved it back to 9.0.